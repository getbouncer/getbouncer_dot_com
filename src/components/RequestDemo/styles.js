import styled from '@emotion/styled'
import { breakpoints } from '~styles'
import gradientBg from '~images/request-demo-background.png'

export const Wrapper = styled.section`
  background-image: url(${gradientBg});
  background-repeat: no-repeat;
  background-position: left -100px;
  padding: 120px 32px 130px;
  text-align: center;

  @media ${breakpoints.smartphone} {
    padding: 90px 32px 80px;
  }
`

export const Icon = styled.img`
  width: 72px;
`

export const Headline = styled.h2`
  font-size: 2.857rem;
  margin-bottom: 35px;

  @media ${breakpoints.smartphone} {
    font-size: 48px;
  }
`

export const TextContent = styled.div`
  margin: 0 auto 40px;
  max-width: 466px;
`
