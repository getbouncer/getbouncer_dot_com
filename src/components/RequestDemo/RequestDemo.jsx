import React from 'react'
import CtaButton from '~components/CtaButton'
import { Headline, TextContent, Wrapper } from './styles'

const RequestDemo = () => (
  <Wrapper>
    <Headline>Request a demo</Headline>
    <TextContent>
      We’d love to hear from you. Reach out and we’ll find a time to connect to learn more about your business, specific
      challenges, and walk through a demo customized for you.
    </TextContent>
    <CtaButton text="Request a demo" to="/contact" />
  </Wrapper>
)

export default RequestDemo
