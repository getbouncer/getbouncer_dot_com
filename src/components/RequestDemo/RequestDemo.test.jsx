import React from 'react'
import renderer from 'react-test-renderer'
import { describe, expect, it } from 'jest'
import RequestDemo from './RequestDemo'

describe('RequestDemo', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<RequestDemo />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
