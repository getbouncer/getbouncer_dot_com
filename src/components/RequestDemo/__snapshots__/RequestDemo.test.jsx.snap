// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RequestDemo renders correctly 1`] = `
<section
  className="css-ttxm7f"
>
  <h2
    className="css-12lyth5"
  >
    Request a demo
  </h2>
  <div
    className="css-1y4oxou"
  >
    We’d love to hear from you. Reach out and we’ll find a time to connect to learn more about your business, specific challenges, and walk through a demo customized for you.
  </div>
  <a
    className="css-e1lh64"
    href="/contact"
  >
    Request a demo
  </a>
</section>
`;
